content/includes/footer.html

```diff
@@ -104,6 +104,11 @@ <h4 class="va-crisis-line">Get help from Veterans Crisis Line</h4>
     dataLayer = [{
       'environment': '{{ buildtype }}'
     }];
+    // We want to know if we are being iframed
+    if ( self !== top ) {
+      console.log('We are being iFramed : ' + self + ' : ' + top);
```

Comment:
Please remove the console.log since there are versions of IE (old ones) that will crash.  Also don't want the console.log in prod code.